alma_api_cli/cli.py

```diff
@@ -0,0 +1,10 @@
+import click
+
+
+@click.group()
+def cli():
+    pass
+
+
+if __name__ == "__main__":
```

Comment:
I don't think you need this block if we're installing and running it as a script, and I would remove it to encourage people to do that :)